docs/source/reservoir-evaporation-algorithm.rst

```diff
@@ -0,0 +1,113 @@
+###################################
+OpenDCS Reservoir Evaporation Algorithm
+###################################
+
+****************************
+ResEvapAlgorithm
```

Comment:
Only the first Title needs to be surround, the remaining just need the symbols underneath the section title.